### PR TITLE
Add job_applicants endpoint docs

### PR DESCRIPTION
**Pull request by florencelau**

### CODE REVIEW

File: reference/Gusto-API.v1.yaml [hunk 1]
```diff
@@ -2264,6 +2264,173 @@ paths:
...
+                email:
+                  type: string
+                  minLength: 1
+                phone:
+                  type: string
+                  minLength: 1
+                onboarding_person_type:
+                  type: string
+                  minLength: 1
+                  description: Can be "Employee" or "Contractor". Must be "Employee" if send_offer is set to true.
```
Comment (ChristianAlexander) [inline reference/Gusto-API.v1.yaml:2321]: Consider using an `enum` here to describe the possible options.

File: reference/Gusto-API.v1.yaml [hunk 2]
```diff
@@ -2264,6 +2264,173 @@ paths:
...
+        name: job_applicant_uuid
+        in: path
+        required: true
+        description: The UUID of the job applicant
+    get:
+      summary: Get a job applicant
+      operationId: get-v1-companies-company_id-job_applicants-uuid
+      description: Returns a single job applicant.
+      tags:
+        - Job Applicants (Beta)
```
Comment (ChristianAlexander) [inline reference/Gusto-API.v1.yaml:2286]: Consider adding this tag value to the `tags` list at the top of the file to control the order in which the sections render.

File: reference/Gusto-API.v1.yaml [hunk 3]
```diff
@@ -2264,6 +2264,173 @@ paths:
...
+    put:
+      summary: Update a job applicant
+      operationId: put-v1-companies-company_id-job_applicants-job_applicant_uuid
+      responses:
+        '200':
+          $ref: '#/components/responses/Job-Applicant-Object'
+      description: Update an existing job applicant (only allowed when the job applicant has not been imported).
+      tags:
+        - Job Applicants (Beta)
+      requestBody:
```
Comment (ChristianAlexander) [inline reference/Gusto-API.v1.yaml:2299]: If this request body is the same as the `put` request body, consider defining it in the `requestBodies` component of this document and referencing it there. Having a single location to define the request body can be helpful for long-term maintenance / consistency of the docs.

**Comment by florencelau** [comment]
closing - moved to https://github.com/caseyrollins/api.gusto.dev/pull/24

**Pull request closed without merging**